src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -128,10 +128,18 @@ public HomeWorkspaceViewModel(HomeWorkspaceModel model, DynamoViewModel dynamoVi
             hwm.SetNodeDeltaState +=hwm_SetNodeDeltaState;
 
             dynamoViewModel.Model.ShutdownStarted += Model_ShutdownStarted;
-
+            dynamoViewModel.PropertyChanged += DynamoViewModel_PropertyChanged;
             SetupFooterNotificationItems();
         }
 
+        private void DynamoViewModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == "CurrentSpace" && !(sender as DynamoViewModel).ViewingHomespace)
```

Comment:
use `nameof` to avoid spelling mistakes and make refactoring easier.

Reply:
> use `nameof` to avoid spelling mistakes and make refactoring easier.

Done

Reply:
@QilongTang Kindly, Could you let me know if we have any pending here please ?